spec/quiet_quality/tools/runner_examples.rb

```diff
@@ -148,6 +157,46 @@
           end
         end
       end
+
+      context "when a command_override is supplied" do
+        let(:command_override) { ["x", "y"] }
+
+        context "when there are no changes to consider" do
+          let(:changed_files) { nil }
+          it { is_expected.to eq(command_override) }
+        end
+
+        context "when there are changes to consider" do
+          context "but they are empty" do
+            let(:changed_files) { empty_changed_files }
+            it { is_expected.to be_nil }
+          end
+
+          context "but they contain no files matching the relevance method" do
+            let(:changed_files) { fully_changed_files(irrelevant_name) }
+            it { is_expected.to be_nil }
+          end
+
+          context "and they contain some files that are relevant" do
+            let(:changed_files) { fully_changed_files(relevant_name, irrelevant_name) }
+
+            context "but none matching the file_filter" do
+              let(:file_filter) { /never_gonna_give_you_up/ }
```

Comment:
❤️

Reply:
Just reviewing away and then BAM. CodeRolled.